expo-app/app/components/Header/NextClassComponent.tsx

```diff
@@ -0,0 +1,95 @@
+import { AuthContext } from "@/app/contexts/AuthContext";
+import { GoogleCalendarEvent } from "@/app/utils/types";
+import React from "react";
+import { Text } from "react-native"; // Make sure you import Text for displaying the message
+import { fetchSameDayCalendarEvents } from "@/app/services/GoogleCalendar/fetchingUserCalendarData"; // Assuming the function is exported here
+
+export default function NextClassComponent({ style }: { style?: any }) {
+  const auth = React.useContext(AuthContext);
+  const user = auth?.user || null; // Check if the user is logged in
+  const [nextClass, setNextClass] = React.useState<GoogleCalendarEvent | null>(
+    null
+  );
+  const [timeUntilNextClass, setTimeUntilNextClass] = React.useState<
+    string | null
+  >(null);
```

Comment:
Can we put these into a more readable format? Something like
```suggestion
  const [nextClass, setNextClass] = useState<GoogleCalendarEvent | null>(null);
  const [timeUntilNextClass, setTimeUntilNextClass] = useState<string | null>(null);
```
We would need to import { useState } from "react"